config/environments/test.rb

```diff
@@ -58,13 +57,4 @@
 
   # Annotate rendered view with file names.
   # config.action_view.annotate_rendered_view_with_filenames = true
-
-  # Set a css_compressor so sassc-rails does not overwrite the compressor when running the tests
-  config.assets.css_compressor = nil
-
-  config.after_initialize do
```

Comment:
Was removing `Bullet` a deliberate choice or the effect of the update script?

Reply:
Whoops - have put that back in 😃